Api/ride.py

```diff
@@ -1,35 +1,39 @@
 import json
```

Comment:
Having docstring in the beginning of every module is adviced for code readability.